index.js

```diff
@@ -53,6 +53,7 @@ module.exports = {
     'mocha/no-pending-tests': 'error',
     'mocha/no-skipped-tests': 'warn',
     'mocha/valid-test-description': 'error',
+    'no-template-curly-in-string': 'off',
```

Comment:
Well a bunsen-model can be referenced from a json file or js. I currently turn it off in my application.

I don't really see the value in the rule personally even without the bunsen issue. It's more of a rule that double checks if you meant to use a literal (in which case it should be a `warn`). Ideally, you should have tests that cover it